app/assets/stylesheets/incurred_incidentals/form.css

```diff
@@ -0,0 +1,13 @@
+.multi-upload-desc {
+  width: inherit !important;
```

Comment:
yes they are important, i dont remember why exactly but it will be the wrong width if i dont.